fast_hdbscan/tests/test_layer_clusters.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from scipy.spatial import distance
+from scipy import sparse
+from scipy import stats
+from sklearn.utils.estimator_checks import check_estimator
+from sklearn.utils._testing import (
+    assert_array_equal,
+    assert_array_almost_equal,
+)
+from fast_hdbscan import (
+    LayerClustering,
+)
+from fast_hdbscan.hdbscan import clusters_from_spanning_tree
+
+# from sklearn.cluster.tests.common import generate_clustered_data
+from sklearn.datasets import make_blobs
+from sklearn.utils import shuffle
+from sklearn.preprocessing import StandardScaler
+from scipy.stats import mode
+
+from tempfile import mkdtemp
+from functools import wraps
+import pytest
+
+from sklearn import datasets
+
+import warnings
+
+n_clusters = 3
+# X = generate_clustered_data(n_clusters=n_clusters, n_samples_per_cluster=50)
+X, y = make_blobs(n_samples=200, random_state=10)
```

Comment:
I know it is the default but I would rather explicitly pass `centers=n_clusters` here.